src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -498,6 +549,95 @@ public TripMonitorNotification checkTripForDelay(NotificationType delayType) {
         return null;
     }
 
+    /**
+     * Check for major upcoming change and notify the traveler.
+     */
+    @Nullable
+    public TripMonitorNotification checkTripForMajorChange(
+        NotificationType delayType,
```

Comment:
Rename this argument to `notificationType`.

Reply:
Refactored and addressed in: https://github.com/ibi-group/otp-middleware/pull/275/commits/91d66190cb611283993b8e745148641efd695c5e